app/utils/report/templates/test.txt

```diff
@@ -18,7 +18,6 @@ Test results summary
 {%- set print_header = true %}
 {%- for t in test_groups|sort(attribute='name') %} {# test_groups #}
   {%- if t.total["FAIL"] or t.regressions %} {# fail or regressions #}
-    {%- set failures = true %}
```

Comment:
OK, how about squashing this with the first commit?  It's not really helpful to have this in the history.

Reply:
Oh, for some reason I thought it can be squashed while merging to the main repository. Nonetheless, I pushed the sqashed commit.